pkg/cmd/pr/list/http.go

```diff
@@ -13,7 +13,7 @@ func shouldUseSearch(filters prShared.FilterOptions) bool {
 	return filters.Draft != nil || filters.Author != "" || filters.Assignee != "" || filters.Search != "" || len(filters.Labels) > 0
 }
 
-func listPullRequests(httpClient *http.Client, repo ghrepo.Interface, filters prShared.FilterOptions, limit int) (*api.PullRequestAndTotalCount, error) {
+func ListPullRequests(httpClient *http.Client, repo ghrepo.Interface, filters prShared.FilterOptions, limit int) (*api.PullRequestAndTotalCount, error) {
```

Comment:
🤔 Since this is now exported and used by multiple commands, I wonder if it should be extracted into a file like https://github.com/cli/cli/blob/trunk/api/queries_pr.go

We did something similar in #9721 , [migrating `listLicenseTemplates` to `api.RepoLicenses`](https://github.com/cli/cli/pull/9721/files#diff-1383afb03d1730b7f23e2da529e535aa1fc1c5ac500fc802899dafa66fcbb61eL247-R247)

What do you think?